plugins/chat/spec/system/shortcuts/mark_all_read_spec.rb

```diff
@@ -15,8 +16,9 @@
     sign_in(user_1)
     Fabricate(:chat_message, chat_channel: channel_1)
     Fabricate(:chat_message, chat_channel: channel_1)
-    Fabricate(:chat_message, chat_channel: channel_2)
-    Fabricate(:chat_message, chat_channel: channel_2)
+    10.times do |i|
+      Fabricate(:chat_message, chat_channel: channel_2, message: "all read message #{i}")
```

Comment:
I think there exists:

```
Fabricate.times(4, :company, name: 'Hashrocket') do
  type 'consultancy'
end
```

but I'm not sure how different it is from what you're doing here.

Reply:
Oh that's interesting I haven't seen that, will give it a try

Reply:
Ah only downside is that you can't access the number, so will leave it for this time